pkgs/applications/misc/mlterm/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
     harfbuzz fribidi m17n_lib openssl libssh2
   ];
 
+  patches = [ ./x_shortcut.c.patch ]; #fixes numlock in 3.7.2. https://bitbucket.org/arakiken/mlterm/commits/4820d42c7abfe1760a5ea35492c83be469c642b3
+
```

Comment:
If the patch can't be used directly from upstream, maybe add a note here that it has to be removed for next point release?